system-int-test/test/cmr/system_int_test/search/collection_creation_search_test.clj

```diff
@@ -0,0 +1,89 @@
+(ns cmr.system-int-test.search.collection-creation-search-test
+  "Integration test for searching collections created after a given date.
+   These tests are to ensure proper CMR Harvesting functionality."
+  (:require
+   [clj-http.client :as client]
+   [clojure.test :refer :all]
+   [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.dev-system-util :as dev-system-util]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.search-util :as search]))
+
+(use-fixtures :each (join-fixtures
+                      [(ingest/reset-fixture {"provguid1" "PROV1"})
+                       (dev-system-util/freeze-resume-time-fixture)]))
+
+(defn- current-time
+  []
+  (str (first (clojure.string/split (str (java.time.LocalDateTime/now)) #"\.")) "Z"))
+
+(deftest search-for-new-collections
```

Comment:
Now that we are supporting multiple `created_at` search parameter and with AND options, we should add more tests to cover the api. Had you have tests for AND options, you would have discovered the bug on options support which I commented earlier.